plugins/scaffolder/src/components/TaskPage/TaskPageLinks.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { parseEntityName } from '@backstage/catalog-model';
+import { IconComponent, IconKey, IconLink, useApp } from '@backstage/core';
+import { entityRoute } from '@backstage/plugin-catalog-react';
+import { Box } from '@material-ui/core';
+import LanguageIcon from '@material-ui/icons/Language';
+import { generatePath } from 'react-router';
+import { TaskOutput } from '../../types';
+
+type TaskPageLinksProps = {
+  output: TaskOutput;
+};
+
+export const TaskPageLinks = ({ output }: TaskPageLinksProps) => {
+  const { entityRef, remoteUrl } = output;
+  let { links = [] } = output;
+  const app = useApp();
+
+  const iconResolver = (key: IconKey | undefined): IconComponent =>
+    key ? app.getSystemIcon(key) ?? LanguageIcon : LanguageIcon;
+
+  if (remoteUrl) {
+    links = [{ url: remoteUrl, title: 'Repo' }, ...links];
+  }
+
+  if (entityRef) {
+    links = [
+      {
+        url: generatePath(
+          `/catalog/${entityRoute.path}`,
```

Comment:
Can't make this assumption here, we need to use the route ref from `@backstage/catalog-plugin-react` when navigating to the entity